packages/libclc.rb

```diff
@@ -51,7 +51,7 @@ def self.patch
   def self.build
     @cmake_options = case ARCH
                      when 'i686', 'x86_64'
-                       CREW_CMAKE_FNO_LTO_OPTIONS.gsub('-fno-lto', '')
+                       CREW_CMAKE_OPTIONS.gsub('-flto=auto', '').sub('-DCMAKE_INTERPROCEDURAL_OPTIMIZATION=TRUE', '')
```

Comment:
Why can't we make `no_lto` work for this sort of thing?

Reply:
This behaviour isn't what `no_lto` normally does. `no_lto` will cause `-fno-lto` to be passed instead of `-flto=auto`, but this is just removing `-flto=auto`. 

I'm not exactly sure why this is being done, so I didn't change it to normal.

Reply:
Can we change no_lto to both disable lto and also disable flto=auto?

Reply:
`flto=auto` is lto, not sure what you mean.

Reply:
I mean that `no_lto` should remove all added `lto` lines such as `-flto=auto` and also set the other lto lines such as the interprocedural cmake option to false. We should not have to edit cmake_options at the package level for every package. Just putting `no_lto` in the package file should be sufficient.

Reply:
`no_lto` does that. These packages have other behavior, which is why `no_lto` does not work for them. As you added the packages, you have the best understanding of why they do this and if it is necessary.

Reply:
I'm just saying that we should make things so that in package files we never need to adjust for no_lto beyond saying no_lto...

Reply:
The custom modifications are from before we had a functional no_lto...

Reply:
Ah, I see.